README.md

```diff
@@ -1,23 +1,145 @@
 # Microsoft Graph Training Module - Using Microsoft Graph .NET SDK to interact with Outlook Calendar

```

Comment:
Just to confirm, this is not going to be used as a lab, correct?

Reply:
@MIchaelMainer to the best of my knowledge this is a standalone sample, unless there are plans to add it to an exisitng lab.